hypervisor/arch/x86/Kconfig

```diff
@@ -4,12 +4,16 @@ config SCENARIO
 	help
 	  By specifying SCENARIO name, ACRN will load corresponding VM configurations
 	  to build the hypervisor. Currently ACRN provides below reference scenarios:
-	  industry: Typical scenario for industry usage with 8 VMs: one pre-launched SOS VM,
+	  - industry: Typical scenario for industry usage with 8 VMs: one pre-launched SOS VM,
```

Comment:
> I fixed some missing commas, misspelling, added note about hyperthreading, and fixing proper reST formatting for bullet lists (blank line before first item, indenting):
> 
> ```
> help
>       By specifying SCENARIO name, ACRN will load corresponding VM configurations
>       to build the hypervisor. Currently, ACRN provides four reference scenarios:
> 
>       - industry: Typical scenario for industry usage with 8 VMs: one pre-launched SOS VM,
>         one post-launched KATA VM, one post-launched RT VM for real-time control,
>         and up to five post-launched Standard VMs; requires 8 virtual cores (e.g., four cores
>         with hyperthreading enabled in the BIOS)
>       - hybrid: Typical scenario for hybrid usage with 3 VMs: one pre-launched Safety VM,
>         one pre-launched Service VM and one post-launched Standard VM;
>       - hybrid_rt: Typical scenario for hybrid real-time usage with 3 VMs: one
>         pre-launched RTVM, one pre-launched Service VM, and one post-launched Standard VM;
>       - logical_partition: Typical scenario that runs two isolated pre-launched VMs
> 
>       Other values are possible if a corresponding scenario has been created
>       under 'misc/vm_configs/scenarios/' in the source code.
> ```

I will correct this (I don't think it's strictly needed since this is not ReST text, it's only seen when looking for help for that option in `make menuconfig`). AFAIK we do not use that text anywhere in our documentation (although a good idea may be to retrieve this to populate this page: https://projectacrn.github.io/latest/reference/kconfig/index.html

Reply:
This help text is retrieved and **does** show up in our documentation:  https://projectacrn.github.io/latest/reference/kconfig/CONFIG_SCENARIO.html#help

Reply:
Oh wow, I had completely missed that... nice!! :-)